### PR TITLE
add operations compatible with pytorch 1.8

### PR DESCRIPTION
**Pull request by underdogliu**
This is in order to address the in-compatibility issue #15. I made following changes and observations:

- Add operations which use FFT utilities from pytorch 1.8. Run test passed and the per-dim error is within `10^-8`. 
- Keep the original 1.7.1 operations, which will be called by a version checking.

@zh217 This is just a quick fix on in-compatibility issue. In future I think we can create version control of the modules, just like torchaudio (its >=0.7.2 will only be compatible with pytorch 1.7.1 while earlier versions are associated with only earlier versions of pytorch. I think we can have such as well)

### CODE REVIEW

**Comment by eddiebergman** [comment]
Any update one this?

After inspecting the log, the builds fail when trying to import `import torch.fft` which makes sense when running older versions of torch. A quick workaround would just to guard the import with a `try ... catch ...` but an if statement on the version is a bit more specific.
```Python
try:
    import torch.fft
except ImportError:
    pass
```
